src/solvers/refinement/string_refinement.cpp

```diff
@@ -312,7 +312,11 @@ bool is_char_pointer_type(const typet &type)
 
 /// \param type: a type
 /// \param ns: name space
-/// \return true if a subtype is an pointer of characters
+/// \return true if a subtype of `type` is an pointer of characters.
+///         The meaning of "subtype" is in the algebraic datatype sense:
```

Comment:
May be being dense, but I don't completely get this, perhaps an example (e.g. a pointer to a char or struct with a char component)